src/common/components/Input/index.tsx

```diff
@@ -145,15 +145,13 @@ export class Input extends React.Component<InputProps, InputState> {
                     <InputLabel htmlFor={this.state.id} error={showError}>
                         {label}
 
-                        {description && (
-                            <Description description={description} />
-                        )}
+                        {description && <Description description={description} />}
                     </InputLabel>
                 )}
 
                 <InputElement id={this.state.id} onBlur={this.makeDirty} hasError={showError} aria-invalid={showError} {...rest} />
 
-                <Collapse isOpen={showError && typeof error === 'string'} className={`${inputErrorTransition}`}>
```

Comment:
Den här string-checken vill vi inte ta bort, den skulle isf göra så att Collapse-grejen visas även när man skickar in error som en boolean, vilket skulle resultera i ett tomt felmeddelande. :)

Reply:
yes